src/components/calculator.js

```diff
@@ -0,0 +1,43 @@
+function Calculator() {
+  return (
+    <div className="calculator-structure">
+      <div className="output">
+        <div className="current-operand" />
+        <span className="calc-output">0</span>
+      </div>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="colored-btn">
+        ÷
+      </button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="colored-btn">
+        x
+      </button>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="colored-btn">
+        -
+      </button>
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="colored-btn">
+        +
```

Comment:
- [ ] At this stage of your learning, you are encouraged to use class-based components so that you can get familiar with them. Please kindly ensure that the Calculator component is class-based